Dockerfile

```diff
@@ -81,7 +81,11 @@ RUN cd /usr/src/lxc \
 	&& ldconfig
 
 # Install Go
-ENV GO_VERSION 1.4.2
+ENV GO_VERSION 1.5
+RUN curl -sSL https://golang.org/dl/go1.4.2.src.tar.gz | tar -v -C /usr/local -xz \
+	&& cd /usr/local/ && mv go go1.4.2
+RUN cd /usr/local/go1.4.2/src/ && ./make.bash
+ENV GOROOT_BOOTSTRAP /usr/local/go1.4.2
```

Comment:
I agree with @tianon i'd rather use the release binaries directly unless it's absolutely necessary.